src/extensions/unit/unitext_hooks.cpp

```diff
@@ -680,6 +681,285 @@ DECLARE_PATCH(_UnitClass_Per_Cell_Process_AutoHarvest_Assign_Harvest_Mission_Pat
 }
 
 
+/**
+ *  Helper function.
+ *  Creates a unit based on an already existing unit.
+ *  Returns the new unit if successful, otherwise null.
+ *
+ *  @author: Rampastring
+ */
+UnitClass* Create_Transform_Unit(UnitClass* this_ptr) {
```

Comment:
Curley brackets on their own line, and a space between the type names and `*` please.